config/scan_images_sample_config.yml

```diff
@@ -0,0 +1,13 @@
+ECR:
```

Comment:
same convention should be followed .. make it all small

Reply:
resolved